### PR TITLE
calculator which takes input from user using C

### PR DESCRIPTION
**Pull request by ApoorvAgarwal21**
it is a calculator which takes input of two numbers as well as input of operation which user wants to perform.

This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

File: calculator which takes input from user using C [hunk 1]
```diff
@@ -0,0 +1,45 @@
+//to take input from user of the operation to be performed using switch case
```
Comment (MJ10) [inline calculator which takes input from user using C]: Please move this file to folder `NumberAddition` and rename it to your github username

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by ApoorvAgarwal21** [comment]
Sir I m new to within and don't know many things about it that how to use .
Sir please don't mark it spam. Sorry sir but can you tell me how to do it
or can you please do it.

**Comment by MJ10** [comment]
@ApoorvAgarwal21 I have mentioned what changes have to be done, please make those changes, commit them and push them.

**Pull request closed without merging**